azext_iot/common/shared.py

```diff
@@ -17,36 +26,80 @@ class SdkType(Enum):
     dps_sdk = 5
 
 
+# pylint: disable=too-few-public-methods
 class EntityStatusType(Enum):
+    """
+    Resource status.
+    """
     disabled = 'disabled'
     enabled = 'enabled'
 
 
+# pylint: disable=too-few-public-methods
 class SettleType(Enum):
+    """
+    Result type of operation.
```

Comment:
This enum defines how to settle a C2D message.

Reply:
Updated in commit: https://github.com/Azure/azure-iot-cli-extension/pull/4/commits/087bb44c55bb60ddbbac8fe1917adf42827a6a2b